fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java

```diff
@@ -494,6 +495,24 @@ private void analyzeSubquery(Analyzer analyzer) throws UserException {
         queryStmt.setFromInsert(true);
         queryStmt.analyze(analyzer);
 
+        // deal with this case: insert into tbl values();
+        // should try to insert default values for all columns in tbl if set
+        if (isValuesOrConstantSelect) {
```

Comment:
please add the same feature for nereids.(the new planner) You can "set enable_nereids_planner=true" and "set enable_nereids_dml=true" to check the current behavior in nereids